pkg/api/handlers/generic/images.go

```diff
@@ -305,7 +305,8 @@ func GetImages(w http.ResponseWriter, r *http.Request) {
 		utils.Error(w, "Something went wrong.", http.StatusInternalServerError, errors.Wrap(err, "Failed get images"))
 		return
 	}
-	var summaries = make([]*handlers.ImageSummary, len(images)+1)
+	// +1 removed from len(images) as this leaves a null entry at the end of the image array
```

Comment:
No need for this comment.  It will mean nothing in the future.  Please remove the comment and just ad it to the PR.

Reply:
Removed comment